### PR TITLE
feat: [modal migration] shortcuts

### PR DESCRIPTION
**Pull request by el**
## Changes

Screenshot 2022-11-18 at 17 22 41
Screenshot 2022-11-18 at 17 22 46

### CODE REVIEW

File: packages/extension/src/newtab/ShortcutLinksModal.tsx
Comment (rebelchris) [inline packages/extension/src/newtab/ShortcutLinksModal.tsx:72]: I changed this in a separate PR, which blocks this for now.

**Review by rebelchris — CHANGES_REQUESTED** [review]

**Review by sshanzel — APPROVED** [review]
Other than what Chris has mentioned, the cleanup looks amazing to me. Always so satisfying to see every time a modal is converted ✨ ✨ ✨ 

With me approving, I think (and hope) merging is still blocked as change is requested by another reviewr. But let me test it now 😆

**Comment by rebelchris** [comment]
Found one more concern, this is already broken, but let's fix it before merging this PR.

**Comment by rebelchris** [comment]
@sshanzel @el I also revamped the Most Visited sites modal as it was part of this PR.

Here is the new one:
Screenshot 2022-11-22 at 11 51 53

**Review by sshanzel — APPROVED** [review]
Thanks for updating the accepting of permission modal as well! 🚢

**Comment by el** [comment]
Thanks for taking care of this one @rebelchris  ❤️

**Review by rebelchris — APPROVED** [review]
Let's do it 🛳️

**Pull request merged**
